ceph_deploy/tests/test_cli_mon.py

```diff
@@ -65,11 +63,11 @@ def test_simple(tmpdir, capsys):
 """)
 
     ns = argparse.Namespace()
-    ns.pushy = mock.Mock()
-    conn = mock.NonCallableMock(name='PushyClient')
+    ns.pushy = Mock()
```

Comment:
I completely agree.  That's part of my goal for test cleanup here, and one reason why I called it "round one".  haha.  Just wanted to get things into a consistent place first.

Interestingly, I did play with this exact test a bit first, and on first try the things I tried to remove related to mocking the push stuff made things fail (in ways I though weird, given that it doesn't exist).  But I decided I would come back to it later.